Postman/Survey/SampleData.csv

```diff
@@ -0,0 +1 @@
+;;;;;
```

Comment:
Should we only test with an empty CSV?

Reply:
We would need a CVS with specific IDs, so to share with Secura we provide an empty one to know that they need a file to test.